src/components/tl/mlx5/tl_mlx5.c

```diff
@@ -28,6 +28,19 @@ static ucc_config_field_t ucc_tl_mlx5_lib_config_table[] = {
      ucc_offsetof(ucc_tl_mlx5_lib_config_t, dm_buf_num),
      UCC_CONFIG_TYPE_ULUNITS},
 
+    {"FORCE_REGULAR", "y",
```

Comment:
can you encapsulate all these three env vars into one with 3 options

Reply:
The three options are orthogonal:
- FORCE_REGULAR: makes width and height divisors of ppn
- FORCE_LONGER: makes h <= w
- FORCE_WIDER: makes w <= h
In particular, to impose square blocks, i.e. w=h, we need to turn on both options